controllers/user.controller.js

```diff
@@ -0,0 +1,60 @@
+var passport = require('passport');
+var strategy = require('passport-facebook');
+const env = process.env.NODE_ENV || 'development';
+const config = require('../config/facebook_oauth.js')[env];
+const db = require('../models');
+const FacebookStrategy = strategy.Strategy;
+
+// Serialize data into the session. sending only the user id
+passport.serializeUser(function(user, done) {
+    done(null, user.dataValues.id);
```

Comment:
I think you can access `user.id` directly

Reply:
Fixed